content/yaml-code-signing/code-signing-identities.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Code signing identities
+description: How to set up code signing identities
+weight: 4
+aliases: /code-signing-yaml/code-signing-identities
+---
+
+Teams can use code signing identities to manage their code signing files in **Team settings > codemagic.yaml settings > Code signing identities** and reference them from their `codemagic.yaml` configuration.
+
+## Managing and uploading files
+
+Team owner permissions are required to upload and edit files under the **Code signing identities** section. However, all team members can view the file info for any of the uploaded files.
+
+### Apple certificates
+
+Codemagic lets you upload certificates with the `.p12` or `.pem` extension. When uploading, Codemagic will ask you to provide the certificate password along with a unique **Reference name**, which can then be used in the `codemagic.yaml` configuration to fetch the specific file.
+
+In addition, Codemagic provides the option to generate a new `Development` or `Distribution` certificate. Click on **Generate certificate**, provide a **Reference name** and choose the certificate type and a matching bundle identifier. Once the certificate has been created, Codemagic will allow you to download the certificate and provides the password for it. The download is available only once.
+
+Note that Apple limits the number of `Distribution` certificates to 3. If you have already reached the maximum number of certificates, the following error will be displayed:
+
+```
+There is a problem with the request entity - You already have a current Distribution certificate or a pending certificate request.
+```
+
+Similar errors can also arise in rarer cases with `Development` certificates. To resolve the error, either remove some old certificate or upload an existing certificate manually.
+
+However, in the case of existing certificates generated by Codemagic, it is possible to automatically fetch them from the Apple Developer Portal based on the bundle identifier.
```

Comment:
based on the API key

Reply:
There should be a notebox about why it's not possible to fetch and use some of the certs that are available in ADP.